pgxn/neon/walproposer.c

```diff
@@ -2162,7 +2211,7 @@ AsyncReadMessage(Safekeeper *sk, AcceptorProposerMessage *anymsg)
 		}
 	}
 	wp_log(FATAL, "unsupported proto_version %d", wp->config->proto_version);
-	return false; /* keep the compiler quiet */
+	return false;				/* keep the compiler quiet */
```

Comment:
Probably a spurious change?

Reply:
I run `make neon-pgindent` from time to time (unfortunately still don't have it in CI), likely it's trace.